### PR TITLE
[simdutf] Update to 5.3.0

### PR DESCRIPTION
**Pull request by trflynn89**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by trflynn89** [comment]
@microsoft-github-policy-service agree

File: ports/simdutf/portfile.cmake [hunk 1]
```diff
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO simdutf/simdutf
     REF "v${VERSION}"
-    SHA512 cd967640ce7a587ba7d0fbf1311bbced04beab341c03875f541179619e561773dffc8feb4b4e685ef549e7ae915b2bbd67f8b0100fd354d77ff0e068b896593f
+    SHA512 b4d1d50fcdb801a6c9b98869a3511ebc95bccb5b8ab08c60d8f9b27bd132b15b66c8a344d97327035c2800bab74a7889253d6e496d3d36b8f6cdab8d10ad58c6
     HEAD_REF master
 )
```
Comment (jimwang118) [inline ports/simdutf/portfile.cmake:8]: It is recommended to add the fastbase64 executable program that needs to be cleaned in vcpkg_copy_tools.

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
All features passed with following triplets:

```
x86-windows
x64-windows
```

**Pull request merged**